fedimint-api/Cargo.toml

```diff
@@ -41,7 +41,7 @@ gloo-timers = { version = "0.2.5", features = ["futures"] }
 wasm-bindgen-futures = "0.4.33"
 
 [target.'cfg(not(target_family="wasm"))'.dependencies]
-tokio = { version = "1.23.0", features = ["rt", "rt-multi-thread", "sync", "time", "signal"] }
+tokio = { version = "1.23.0", features = ["full"] }
```

Comment:
At some point we need to review features, for tokio `full` is probably ok for now because we use it everywhere and for all kinds of things, but ideally we could track which features we actually use.